tests/features/common/components/date-picker.component.js

```diff
@@ -0,0 +1,89 @@
+import { By } from 'selenium-webdriver'
+import { locatorBuilder } from '../../common-tools/common-tools'
+
+import _ from 'lodash'
```

Comment:
It's more performant to import only the specific methods you use from Lodash, especially when it's just a few.

```suggestion
import { cloneDeep } from 'lodash'
```